drivers/ethernet/eth_liteeth.c

```diff
@@ -96,7 +99,10 @@ static int eth_tx(const struct device *dev, struct net_pkt *pkt)
 
 	/* wait for the device to be ready to transmit */
 	while (sys_read8(LITEETH_TX_READY) == 0) {
-		;
+		if ((attempts++) == MAX_TX_FAILURE) {
```

Comment:
I don't think we normally use parens around increment in such syntactic context in Zephyr.

Reply:
I removed parnes as suggested